tests/test-webapps/test-jetty-webapp/src/main/java/com/acme/CookieDump.java

```diff
@@ -117,9 +115,9 @@ protected String deScript(String string)
     {
         if (string==null)
             return null;
-        string=string.replace("&", "&amp;");
-        string=string.replace( "<", "&lt;");
-        string=string.replace( ">", "&gt;");
+        string = string.replace("&", "&amp;");
+        string = string.replace( "<", "&lt;");
+        string = string.replace( ">", "&gt;");
```

Comment:
another candidate for a trie replace!

Reply:
I didn't change this code to use `StringUtil`, as it's part of the `com.acme` tree and I didn't want to introduce a jetty-util dependency for that.